src/intellisense/wrapper.rs

```diff
@@ -1,17 +1,18 @@
+use com::Interface;
+
 use crate::intellisense::ffi::*;
 use crate::os::{CoTaskMemFree, BSTR, LPSTR};
 use crate::utils::Result;
 use crate::wrapper::Dxc;
-use com_rs::ComPtr;
 use std::ffi::CString;
+use std::mem::ManuallyDrop;
 
-#[derive(Debug)]
 pub struct DxcIntellisense {
-    inner: ComPtr<IDxcIntelliSense>,
+    inner: IDxcIntelliSense,
```

Comment:
@MarijnS95 it seems odd to not have the ComPtr RAII wrapper type anymore, is this really how the new com crate expects you to do this? Or is it just missing the smart pointer type?

Reply:
They dropped the smart pointer type in favour of having the "interface" be a struct that _owns_ the COM object internally. Drop the struct, causes the refcount to be decremented.